utils/aws/s3.ts

```diff
@@ -0,0 +1,41 @@
+import aws from "aws-sdk";
+import fs from "fs";
+import path from "path";
+
+const S3 = new aws.S3();
+
+/* 
+
+Uploads a file to the AWS S3 bucket corresponding to AWS_BUCKET_NAME in .env.
+    * filePath: the path of the file to upload
+    * returns: upload response if successful, error object otherwise
+
+Sample usage:
+    myFile = "~/documents/my_file.txt"
+    res = uploadToS3(myFile)
+
+*/
+
+const uploadToS3 = (filePath: string): void => {
```

Comment:
the function shouldn't be returning void - it should maybe return some type of response object that indicates if the upload was successful, error messages, and potentially the data object

Reply:
I think it would be most useful to return the response from the s3 upload.